docs/en/Remote-Logging.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: global
+title: Remote Logging
+group: Features
+priority: 5
+---
+
+* Table of Contents
+{:toc}
+
+## Overview
+Alluxio supports sending logs to a remote log server over the network. This feature can be useful
+to system administrators who have to perform the task of log collection. With remote logging, the
+log files, e.g. master.log, worker.log, etc. on all Alluxio servers will be readily available on
+a designated and configurable directory on the log server.
+
+## Configure Alluxio
+Suppose the hostname of the log server is AlluxioLogServer, and the port is 45010.
+Refer to [Running Alluxio on a cluster](Running-Alluxio-on-a-Cluster.html) for instructions in deploying
+Alluxio in a cluster.
+
+By default, remote logging is not enabled. To enable Alluxio remote logging, you can either set a
+few environment variables or modify the JVM properties directly.
+
+This section describes these two different approaches step by step. There is no requirement on where
+the log server must run, as long as the other Alluxio servers have access to it. In our example, we
+run the log server on the same machine as the primary master. If you want to run the log server on
+another machine, you need to copy the Alluxio directory to the desired machine (with `rsync`
```

Comment:
I don't think it's necessary to specify that Alluxio needs to be installed before you can run its services - anyone reading this guide should know that already.

Reply:
Do we need to emphasize that the alluxio directory on all the machines should have the same `conf` and `libexec` directory? I think sometimes, users may forget syncing the folders across the machines.

Reply:
I thought only the log server port needs to match. As long as we don't break log server backwards compatibility between versions, I could imagine users upgrading their masters and workers to a new version without upgrading their log server.

Reply:
Yeah, we support that.